addOns/graphql/src/main/resources/org/zaproxy/addon/graphql/resources/Messages.properties

```diff
@@ -182,6 +190,10 @@ graphql.engine.strawberry.docsUrl = https://github.com/strawberry-graphql/strawb
 graphql.engine.strawberry.name = Strawberry
 graphql.engine.strawberry.technologies = Python
 
+graphql.engine.tailcall.docsUrl = https://tailcall.io
```

Comment:
Was this meant to be `https://tailcall.run/`?

Reply:
Yup